### PR TITLE
Feat 4468 add python deepgram language detection

### PR DESCRIPTION
**Pull request by Azanul**
Closes https://github.com/appwrite/appwrite/issues/4468

### CODE REVIEW

**Comment by stnguyen90** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

File: python/deepgram-language-detection/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+ℹ️ _Create your Deepgram API key at console.deepgram.com_
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```bash
+git clone https://github.com/open-runtimes/examples.git && cd examples
+cd python/deepgram-audio-summary
```
Comment (adityaoberai) [inline python/deepgram-language-detection/README.md:37]:
```suggestion
cd python/deepgram-language-detection
```

File: python/deepgram-language-detection/main.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+def main(req,res):
+  try:
+    #fileUrl and DEEPGRAM_API_KEY is required
+    file_url = req.payload['fileUrl']
+    api_key = req.variables.get('DEEPGRAM_API_KEY')
+
+  except Exception:
+    return errorMessage(res, 'Payload must contain fileUrl and DEEPGRAM_API_KEY.')
```
Comment (adityaoberai) [inline python/deepgram-language-detection/main.py:19]:
```suggestion
    return errorMessage(res, 'Request must contain the File URL in the payload and the Deepgram API Key as a variable.')
```

The API Key is technically not a part of the request payload, but a part of the variables, so I recommend the change.

**Review by adityaoberai — CHANGES_REQUESTED** [review]
Function mostly looks good.
Just some minor changes and then it will be good to go.

**Review by Meldiron — APPROVED** [review]

**Comment by Meldiron** [comment]
Thanks for the amazing PR 😇 I have tested it and all works great.
Merging the PR now 🥳

**Review by adityaoberai — APPROVED** [review]

**Pull request merged**